### DISCUSSION TITLE
Is Tweepy working?

### DISCUSSION THREAD

**Discussion by AhmadGhachim** [tweepy/tweepy#2105] [Questions] [answered]
I’ve been trying to find a reliable way to scrape Twitter data. I’ve tried using libraries like twitter-scraper, Snscrape Twitter, and Twint, but I’ve run into issues with all of them. I’m considering signing up for a Twitter Developer Account and using Tweepy instead. Has anyone had success in scrapping tweets using Tweepy?

**Comment by Harmon758** [comment, marked as answer]
Yes
